db/re/item_db.conf

```diff
@@ -99720,6 +99727,60 @@ item_db: (
 	View: 1
 },
 {
+	Id: 13089
+	AegisName: "Faceworm_Leg"
+	Name: "Faceworm Leg"
+	Type: 4
+	Buy: 10
+	Weight: 500
+	Atk: 110
+	Matk: 50
+	Range: 1
+	Slots: 1
+	Job: {
+		Novice: true
+		Swordsman: true
+		Magician: true
+		Archer: true
+		Merchant: true
+		Thief: true
+		Soul_Linker: true
+		Ninja: true
+	}
+	Loc: 2
+	WeaponLv: 3
+	EquipLv: 24
+	View: 1
+	Script: <" bonus bAtkEle,Ele_Poison; ">
+},
+{
+	Id: 13090
+	AegisName: "Faceworm_Queens_Leg"
+	Name: "Faceworm Queen's Leg"
+	Type: 4
+	Buy: 10
+	Weight: 500
+	Atk: 180
+	Matk: 120
+	Range: 1
+	Slots: 2
+		Job: {
+		Novice: true
+		Swordsman: true
+		Magician: true
+		Archer: true
+		Merchant: true
+		Thief: true
+		Soul_Linker: true
+		Ninja: true
+	}
+	Loc: 2
+	WeaponLv: 4
+	EquipLv: 100
+	View: 1
+	Script: <" bonus bInt,3; ">
```

Comment:
the faceworm legs have an autobonus that transfer you to faceworm queen
https://www.youtube.com/watch?v=DIuf0NA4c50